README.md

```diff
@@ -123,21 +123,23 @@ gcloud container clusters delete onlineboutique \
     --project=${PROJECT_ID} --zone=${ZONE}
 ```
 
-## Automated Deployment
+## Use Terraform to provision a GKE cluster and deploy Online Boutique
 
-Automated deployment for the Online Boutique sample application is supported in this repository. Instructions for using Terraform to replicate the [**Quickstart (GKE)**](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/README.md#quickstart-gke) steps can be found inside [/terraform/README.md](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/terraform).
+The [`/terraform` folder](terraform) contains instructions for using [Terraform](https://www.terraform.io/intro) to replicate the steps from [**Quickstart (GKE)**](#quickstart-gke) above.
 
-## Other Deployment Options
+## Other deployment variations
```

Comment:
**Thought:**
I think, as a user, it'd be ideal to see the full list of deployment options in a single place.
Currently part of the list exists in `README.md` and the other part in `kustomize/README.md`.
I think, eventually, as we keep adding more Kustomize components, the list in `kustomize/README.md` will be exhaustive.
No action needed.